README.md

```diff
@@ -1,5 +1,7 @@
 # R support for Visual Studio Code
 
+[![Badge](https://aka.ms/vsls-badge)](https://aka.ms/vsls)
+
```

Comment:
So far, we don't have any other badges (e.g. version, cicd, license, installs, rating...) in the readme. Personally I'd suggest we either agree on a sensible set of badges or just not have any badges at all.

Reply:
Makes sense to me - I'll remove the badge for now

Reply:
Putting a single badge is perfectly ok to me, just like <https://github.com/Microsoft/vscode-cpptools> 😺

Reply:
I guess this is just a matter of personal preference, so feel free to leave the badge in, if you agree on this :)